### PR TITLE
IDE-1097 Add submit only the ecl without creating an archive

### PR DESCRIPTION
**Pull request by dehilsterlexis**

### CODE REVIEW

**Comment by dehilsterlexis** [comment]
@GordonSmith for review

File: wlib/EclCommand.h [hunk 1]
```diff
@@ -161,6 +161,7 @@ class CEclCommandMixin : public CMessageMap
         UPDATEUI(cui, ID_ECL_CLEARERRORS, m_ecl->SyntaxGetCount() > 0);
         UPDATEUI(cui, ID_ECL_GOTO, bHasContents);
         UPDATEUI(cui, ID_ECL_SEL_GO, bHasSelection);
+        UPDATEUI(cui, ID_GO_SUBMITNOARCHIV, bCanSubmit);
```
Comment (GordonSmith) [inline wlib/EclCommand.h:164]: Should this be:  `ID_ECL_NOARCHIV_GO`?

Comment (dehilsterlexis) [reply wlib/EclCommand.h:164]: I think I have the names backwards when compared to the current naming convention.

**Review by GordonSmith — APPROVED** [review]

File: eclide/EclDlgBuilder.cpp [hunk 2]
```diff
@@ -748,6 +748,11 @@ void CBuilderDlg::OnEclGo(UINT /*uNotifyCode*/, int nID, HWND /*hWnd*/)
     case ID_GO_SUBMITSELECTED:
         m_owner->OnButtonGo(Dali::WUActionRun, true);
         break;
+    case ID_ECL_NOARCHIV_GO:
+    case ID_GO_SUBMITNOARCHIV:
+        m_archive = false;
```
Comment (GordonSmith) [inline eclide/EclDlgBuilder.cpp:753]: Should this be set back to true after submission, or does that happen elsewhere?

**Review by GordonSmith — CHANGES_REQUESTED** [review]

**Comment by dehilsterlexis** [comment]
@GordonSmith for review

File: eclide/EclDlgBuilder.cpp (hunk 2 above)
Comment (GordonSmith) [reply eclide/EclDlgBuilder.cpp:755]: @dehilsterlexis - did you confirm this eitherway?

**Review by GordonSmith — CHANGES_REQUESTED** [review]

**Comment by dehilsterlexis** [comment]
@GordonSmith for review

**Review by GordonSmith — APPROVED** [review]

**Pull request merged**